cypress/integration/auth/auth-admin.spec.js

```diff
@@ -1,11 +1,14 @@
-describe('Visit admin and authenticate', () => {
-  it('Visits admin, fills the login form and login', () => {
+import comp from '../../support/components';
+import { common } from '../../support/texts';
+
+describe('[Auth - admin] Visit admin and authenticate', () => {
+  it('Should visit admin, fills the login form and login', () => {
     cy.loginSuccess();
   });
-  it('Visit admin and provide wrong credentials', () => {
+  it('Should visit admin and provide wrong credentials', () => {
     cy.loginFail();
-    cy.get('.notification')
-      .contains('Wrong email and/or password')
+    cy.get(comp.common.messageBox)
```

Comment:
`comp` - unnecessary abbreviation
```suggestion
    cy.get(components.common.messageBox)
```